### PR TITLE
pylint errors

### PR DESCRIPTION
**Pull request by itsmeolivia**
removing unnecessary imports

### CODE REVIEW

**Comment by nealwu** [comment]
@itsmeolivia Can you pull and rebase? Your change pulled in two other changes here for some reason.

File: official/resnet/imagenet.py [hunk 1]
```diff
@@ -83,11 +83,11 @@ def create_readable_names_for_imagenet_labels():
   (since 0 is reserved for the background class).
 
   Code is based on
-  https://github.com/tensorflow/models/blob/master/inception/inception/data/build_imagenet_data.py#L463
+  https://github.com/tensorflow/models/blob/master/research/inception/inception/data/build_imagenet_data.py
```
Comment (shlens) [inline official/resnet/imagenet.py]: This is deprecated and should instead point to https://github.com/tensorflow/models/blob/master/research/slim/datasets/build_imagenet_data.py

File: official/resnet/imagenet.py [hunk 2]
```diff
@@ -83,11 +83,11 @@ def create_readable_names_for_imagenet_labels():
...
   """
 
   # pylint: disable=g-line-too-long
-  base_url = 'https://raw.githubusercontent.com/tensorflow/models/master/inception/inception/data/'
+  base_url = 'https://raw.githubusercontent.com/tensorflow/models/master/research/inception/inception/data/'
```
Comment (shlens) [inline official/resnet/imagenet.py]: Ditto from above

File: official/resnet/imagenet.py (hunk 1 above)
Comment (nealwu) [reply official/resnet/imagenet.py]: Thanks Jon! Just set up a separate pull request for this: https://github.com/tensorflow/models/pull/2455

**Pull request merged**